Preface/Preface.pillar

```diff
@@ -58,12 +57,13 @@ available at *http://www.deepintopharo.org*
 
 Do not be frustrated by parts of Pharo that you do not immediately understand.
 You do not have to know everything! Alan Knight expresses this as follows:
-
+{{{latex:\begin{quote} }}}
```

Comment:
Yes, I tried it, with unsatisfactory result. It'isn't note, as rendered Notes: Try not to care. ... it's citation, quote of Alan Knight. Thus I used LaTeX quote ambient, so appropriate for this type of text. 
Btw. Kilon and me have (not so) small talk on #pharo about Pillar as tool for writing book. I consider Pillar good tool for writing short texts, but for book? My personal opinion: a reasonable person can recognize when he made a mistake and can change his decision. IMHO decision using Pillar for writing book was one of that mistakes. When we will change this decision, I am ready to help as I can. I'm ready to help even if we will not change that decision,
with gritted teeth I will fight with flaws of Pillar to produce the best book as I can ;) (I'm not expert, but rather well skilled LaTeX user for 15+ years)